unity-client/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/AB/AssetPromise_AB.cs

```diff
@@ -149,7 +149,7 @@ IEnumerator LoadAssetBundle(string finalUrl, Action OnSuccess, Action OnFail)
 
             if (!assetBundleRequest.WebRequestSucceded())
             {
-                Debug.Log($"request failed? {assetBundleRequest.error} ... {finalUrl}");
+                Debug.Log($"Request failed? {assetBundleRequest.error} ... {finalUrl}");
```

Comment:
can we add an `#IFDEF` here to prevent this message in production? Saw it on my console a couple of times.

Reply:
The logger is disabled in production, maybe you're confusing this message with the built-in one that we can't disable.